include/wlr/types/wlr_seat.h

```diff
@@ -13,11 +13,11 @@
  * managed by wlr_seat; some may be NULL.
  */
 struct wlr_seat_client {
-	struct wl_resource *wl_resource;
 	struct wl_client *client;
 	struct wlr_seat *seat;
 
 	// lists of wl_resource
+	struct wl_list binds;
```

Comment:
I don't like a lot that there are two levels of lists. Why not create one `wlr_seat_client` per bind?

Reply:
What do you call two levels of lists? There already are plenty of lists in wlr_seat_client.
On the other hand, creating one wlr_seat_client per bind will make us iterate over all the binds everywhere we want to notify the client about something, that's a much bigger code change that would break compositor compat (that is fine) and feels less practical to use.
Unless there's any reason to treat one bind differently from the other I'd rather keep it as simple as possible from an interface point of view

Reply:
I'll think about it. In the meantime, can you rename it to `resources`? We don't use the term "bind" anywhere.

Reply:
Hmm, looked around and we're not consistent at all. The one that comes up most is `wl_resources` but we have `resources`, `client_resources`... 

I personally don't like to say just resources as everything is resources, a few lines below (thanks github for showing context!!!) are "resources" to pointers, keyboards etc.

Anyway, renaming for now, but this might need a separate discussion.

BTW I think you're thinking too much, this is basically just what I said in #1023 - or did you understand something else in my comment?

Reply:
Yeah, read it too quickly.